website/src/app/kb/automate/docker-compose/readme.mdx

```diff
@@ -34,10 +34,6 @@ services:
       # See https://docs.rs/env_logger/latest/env_logger/ for more information.
       - RUST_LOG=str0m=warn,info
 
-      # Enable or disable masquerading. Default enabled. Disabling this can prevent
-      # the Gateway from reaching other hosts in your subnet or the internet.
```

Comment:
Maybe someone considers that a feature? That the Gateway can only reach localhost if the Gateway and Resource are always paired onto the same system?

Reply:
I'd say adding more firewall rules to block egress traffic is the way to go to achieve that usecase.